README.md

```diff
@@ -877,6 +878,20 @@ Generate a [ronin-payloads] script:
 $ ronin new payload my_payload.rb
 ```
 
+### Archive
+
+Archive files:
+
+```shell
+$ ronin archive -o archived.zip arch1.txt arch2.txt
+```
+
+Archive files using tar format:
+
+```shell
+$ ronin archive -f tar -o archived.tar arch1.txt arch2.txt
```

Comment:
Omit the `.tar` from the output file or change it to another extension like `archive.foo`, to make it obviously that we're explicitly forcing the command to use `tar` output.

Reply:
Also use `file1.txt`, etc, for the input file names.